src/AutoRest.CSharp.V3/Pipeline/Extensions.cs

```diff
@@ -28,6 +28,14 @@ internal static class Extensions
             _ => null
         };
 
+        public static FrameworkTypeReferenceFormat ToFrameworkTypeReferenceFormat(this Schema schema) => schema switch
```

Comment:
This is very similar to GetSerializationFormat method

Reply:
Fixed. Used only GetSerializationFormat.